_perspective-videos/index.cs.md

```diff
@@ -0,0 +1,70 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "Videa o perspektivách webové přístupnosti: Prozkoumejte dopady a výhody pro všechny uživatele"
+title_html: "Videa o perspektivách webové přístupnosti:<br>Prozkoumejte dopady a výhody pro všechny uživatele"
+nav_title: "Přehled videí o perspektivách"
+lang: cs    # Change "en" to the translated-language shortcode
+last_updated: 2023-01-06    # Put the date of this translation YYYY-MM-DD (with month in the middle)
```

Comment:
@Kikisek You probably mean `2024-01-06`? 😉

Reply:
Thank you!